src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -51,6 +59,25 @@ class MemberPermission(OrganizationPermission):
     }
 
 
+class OrganizationGroupSearchViewGetSerializer(serializers.Serializer[None]):
+    visibility = serializers.ListField(
+        child=serializers.ChoiceField(
+            choices=[v[0] for v in GroupSearchViewVisibility.as_choices()]
+        ),
+        required=False,
+    )
+
+    def validate(self, data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
can add tests but i don't believe this function is necessary if you pass choices into `MultipleChoices` field validator.